Problem-322

```diff
@@ -0,0 +1,55 @@
+/*
+Time Complexity- > O(n);
+
+46 out of 182 test cases passed.
+
+Approach: sorted the given array.
+
+Always pick the biggest denomination. and keep on adding the coins 
+add to the result.
+
+
```

Comment:
We might skip some possible cases if we do that
for 11 using 2,5 if I add 5+5 i can't make 11 using 2 So we need to fix this condition